api/app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
          jwt_revocation_strategy: Devise::JWT::RevocationStrategies::Null
 
   # Constants
-  PASSWORD_REGEX = /(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*\-._]).{8,}$/
+  PASSWORD_REGEX = /(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*\-.])/
```

Comment:
El `_` es necesario que lo mantengamos? Porque vi que lo borraste acá pero no sé que están haciendo desde el FE.
Esto ok con cualquiera de las opciones, pero sí que sean consistentes entre BE y FE

Reply:
https://github.com/wyeworks/finder/pull/211/commits/5b52a461f0489f768fca80fd0c2cb9e3f7974235